protocol/src/codec/transaction.rs

```diff
@@ -362,59 +362,14 @@ impl Decodable for SignedTransaction {
 mod tests {
     use super::*;
 
-    use rand::random;
-
     use common_crypto::secp256k1_recover;
 
     use crate::codec::hex_decode;
     use crate::types::{
         AddressSource, Bytes, CKBTxMockByRefAndOneInput, CellDep, CellWithData, Public, Script,
-        SignatureR, SignatureS, TransactionAction, Witness, H160, H256, U256,
+        SignatureR, SignatureS, Witness, H160, H256, U256,
     };
 
-    fn rand_bytes(len: usize) -> Bytes {
```

Comment:
Why is this test removed ? is there a rplace for it ? or is it no longer relevant?

Reply:
They are some assist functions that do not used by unit tests.